### ISSUE TITLE
RRDP cache disk usage increases since updating to 0.14.0

### ISSUE THREAD

**Issue by urscale** [NLnetLabs/routinator#978]
While pretty constant before, since updating from `0.13.0` to `0.14.0`, the disk usage of `/var/lib/routinator/rpki-cache/rrdp` has been continuously increasing, with only some space getting freed occasionally:

<img width="2256" alt="0 14 0_disk_usage" src="https://github.com/user-attachments/assets/1eb50848-99b0-4a9c-9b4f-bdc12cf90e2c">

Routinator has been installed from package on a Debian Bullseye host.

Let me know if there's anything I can do to help debug the issue.

**Comment by partim** [comment]
Thank you for your report!

Looking at our own installation, the `rrdp` directory (which contains the raw RRDP data) has nearly 5 GB while the `stored` directory (which contains the data actually used) has only 1.3 GB, which seems indeed a bit much. Does this roughly correspond with the values you see?

**Comment by urscale** [comment]
Yeah, we have about 1.3 GB in `stored` as well, with the `rrdp` directory between 2.7 GB and 3.3 GB in size.

One thing I noticed is that out of the three installations of Routinator we're running, one has an exceptionally large binary file `/var/lib/routinator/rpki-cache/rrdp/rrdp.ripe.net/21d6592469dbe79feb2922562764fd193170f173229298b9a4443ffb5c282000.bin`, which is using about 1.3 GB on that instance, while that same file only takes up around 530 MB on our other two instances.

Since all three Routinator instances are configured the same way, synching from the same sources, shouldn't one expect whatever is held in that RRDP cache to be of somewhat similar sizes?

**Comment by pr4gmat1q** [comment]
the same problem after updating to 14.0
![image](https://github.com/user-attachments/assets/8d58d49d-f394-433d-9903-76b6022b86f4)

**Comment by urscale** [comment]
Yay, thanks a lot for sorting this out in `0.14.1`!

**Comment by partim** [comment]
We let a pre-release run over Christmas and it looked fine. We are also going to keep an eye on things but feel free to report if it starts misbehaving again.

**Comment by mmuehlenhoff** [comment]
JFTR; we ran into the same issue at Wikimedia and after running running 0.14.1 for five days I can confirm that it resolves the issue in our installation.
